.travis.yml

```diff
@@ -145,3 +145,16 @@ matrix:
         - cd test
         - mbedtools compile -t GCC_ARM -m K64F
         - ccache -s
+
+    - <<: *mbed-tools-test
+      name: "Test import command"
+      env: NAME=test-import-cmd CACHE_NAME=test-import-cmd
+      script:
+        - mbedtools import mbed-os-example-ble
+        - cd mbed-os-example-ble
+        # Checkout the development branch on Blinky
+        - git checkout development
+        - cd BLE_Advertising
+        - mbedtools deploy
```

Comment:
`mbedtools import` runs `deploy` internally right? Do we need this for any other reason?

Reply:
We need to run the deploy after the checkout of the development branch, otherwise we don't get a deployed version of Mbed OS that has CMake support.